geotrek/outdoor/locale/fr/LC_MESSAGES/django.po

```diff
@@ -357,12 +354,41 @@ msgstr "Ajouter site enfant"
 msgid "Add a brother site"
 msgstr "Ajouter un site frère"
 
-msgid "Attached files"
+#, python-format
+msgid ""
+"You can't delete <strong>%(object)s</strong> because it has child outdoor "
```

Comment:
```suggestion
msgid ""
"You can't delete <strong>%(object)s</strong> because it has children courses or sites. To be able to delete a parent site, you need to delete the children first."
msgstr ""
"Vous ne pouvez pas supprimer <strong>%(object)s</strong> car des sites ou parcours enfants lui sont associés. Pour pouvoir supprimer un site parent, il faut au préalable supprimer ses enfants.
```